packages/serial/src/ZWaveSerialPortBase.ts

```diff
@@ -76,8 +76,15 @@ export function isZWaveSerialPortImplementation(
 	);
 }
 
+type write_cb=(err:Error|null|undefined)=>void
+export interface DuplexAlike {
+	// (chunk: any, encoding?: BufferEncoding | undefined, cb?: ((error: Error | null | undefined) => void) | undefined): boolean; (chunk: any, cb?: ((error: Error | null | undefined) => void) | undefined): boolean;
+	write(chunk:Buffer,cb?:write_cb|undefined):boolean;
+} ;
```

Comment:
@sdrsdr can you explain why this is needed? What's your use case?

In any case this feels out of place in this PR, I'd do this separately.

Reply:
I'm not sure how this got here :( It's part of me experimenting with passing non full duplex complaint interface there. You see we're experimenting with WebSockets  an the ws library does not provide a duplex compatible interface so I was trying to use TS to filter down the actually used methods. Did not get me too far as it's casted multiple times to different interfaces. You can totally omit it. Sorry for the inconvenience